packages/vike-server/src/runtime/adapters/handleViteDevServer.ts

```diff
@@ -3,6 +3,9 @@ import type { IncomingMessage, ServerResponse } from 'node:http'
 import { assert } from '../../utils/assert.js'
 import { globalStore } from '../globalStore.js'
 
+/**
+ * If vite dev server returns a response, we need to forward it to our actual server
+ */
 function handleViteDevServer(req: IncomingMessage, res: ServerResponse): Promise<boolean> {
```

Comment:
The naming may be misleading. It's just the vite middlewares, running with `middlewaremode: true`, there isn't actually a vite server listening.